modules/ding_frontend/ding_frontend.info

```diff
@@ -83,7 +80,6 @@ features[image][] = ding_tetiary_medium
 features[image][] = ding_tetiary_small
 features[language][] = en
 features[menu_blocks][] = main_menu_second_level
-features[menu_blocks][] = 1
```

Comment:
How is this related to the case?

Reply:
In latest development branch, structure of changed file matches with default state.